generator/templates/readme.ts

```diff
@@ -26,26 +26,63 @@ To access the Juju API, a connection must be made to either a Juju
 controller or a Juju model.
 
 \`\`\`javascript
-import { connect } from "@canonical/jujulib";
+import { connect } from "./dist/api/client";
```

Comment:
Should this import be from `@canonical/jujulib` as that's how people will consume it?

Reply:
Nice catch!